nomos-services/robustness/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+// crates
+use async_trait::async_trait;
+use overwatch_rs::services::{
+    handle::ServiceStateHandle,
+    relay::RelayMessage,
+    state::{NoOperator, NoState},
+    ServiceCore, ServiceData, ServiceId,
+};
+
+pub struct RobustnessService {}
+
+impl ServiceData for RobustnessService {
+    const SERVICE_ID: ServiceId = "Robustness";
+    type Settings = RobustnessConfig;
+    type State = NoState<Self::Settings>;
+    type StateOperator = NoOperator<Self::State>;
+    type Message = RobustnessMsg;
+}
+
+#[derive(Clone)]
+pub struct RobustnessConfig {}
+
+#[derive(Debug)]
+pub enum RobustnessMsg {
+    /// Inject a new entropy that will be used for configuration update logic
+    SetEntropy(Box<u8>),
+}
```

Comment:
For now it is ok. But at some point when writing the new consensus we would probably have a consensus event system to subscribe to consensus events. I would like for consensus to not be responsible on when to actually call this. @zeegomo WDYT?

Reply:
Yes, we should probably make a `NewEntropy` message available. 
Apart from this, not sure the robustness layer should exist as its own separate service. It only makes sense with a mixnet, couldn't it be an implementation detail of the mixnet service?

Reply:
I agree with @zeegomo , Rosbustness shouldn't be a service in nomos-nodo per se. As it is something just the mixnet needs. I think it should be embedded in the mixnet service itself.

Reply:
I agree. In our code structure, it’s natural to have robustness in the mixnet service. But then I still have a doubt why we need to have the robustness layer separately in the spec (as designed when I, you, and Marcin discussed). When we discussed, I thought the robustness layer will control other configs for other layers in the future, as a mediator of config correlation between layers. But for v1, it is only for mixnet.

Reply:
@zeegomo @danielSanchezQ 
Okay, I don't need to think complicated. The robustness layer doesn't need to a 'service' in this implementation. It can be implemented conceptually and clearly within a mixnet crate implementation. I just removed the robustness service 31362243. Also, I removed unclear commands and added a comment that the mixnet service should subscribe `NewEntropy` from the consensus service.